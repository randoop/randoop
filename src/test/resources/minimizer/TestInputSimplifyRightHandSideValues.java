import org.junit.FixMethodOrder;
import org.junit.Test;
import org.junit.runners.MethodSorters;

@FixMethodOrder(MethodSorters.NAME_ASCENDING)
public class TestInputSimplifyRightHandSideValues {
  // Test simplification of right hand side values.
  @Test
  public void test1() throws Throwable {
    int a = 1;
    int b = 2;
    int c = a + b;

    org.junit.Assert.assertTrue(c == 3);

    int d = 2 * c;

    org.junit.Assert.assertTrue(d == 6);
    org.junit.Assert.assertTrue(d == 7);
  }
}
