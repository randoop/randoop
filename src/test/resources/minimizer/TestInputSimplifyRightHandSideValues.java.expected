import org.junit.FixMethodOrder;
import org.junit.Test;
import org.junit.runners.MethodSorters;

@FixMethodOrder(MethodSorters.NAME_ASCENDING)
public class TestInputSimplifyRightHandSideValuesMinimized {

    // Test simplification of right hand side values.
    @Test
    public void test1() throws Throwable {
        int d = 0;
        org.junit.Assert.assertTrue(d == 7);
    }
}
